File: sign/sign-interface/src/main/java/cn/moonnow/sign/query/SessionQuery.java
package cn.moonnow.sign.query;

import java.io.Serializable;
import java.util.List;

import lombok.Data;

/**
 * 会话 DEV_SESSION 查询类
 */
@Data
public class SessionQuery implements Serializable {

  private static final long serialVersionUID = 1L;

  protected String sessionId; // 会话编号

  protected String accountId; // 账号编号

  protected String loginKey; // 登录标识

  protected Long currentTimes; // 当前时间

  protected String aesKey; // AES密钥

  protected List<String> sessionIdAndin; // 会话编号Andin查询

  protected String accountIdAndeq; // 账号编号Andeq查询

  protected String loginKeyAndeq; // 登录标识Andeq查询

  protected String accountIdAndKeyLike; // 账号编号AndKeyLike查询

  protected String loginKeyOrKeyLike; // 登录标识OrKeyLike查询

  protected String aesKeyOrKeyLike; // AES密钥OrKeyLike查询

}

File: vue-dev-tool/src/view/cn/moonnow/sign/session/SessionQuery.js
export default class SessionQuery {
  constructor () {
    this.sessionId = null // 会话编号
    this.accountId = null // 账号编号
    this.loginKey = null // 登录标识
    this.currentTimes = null // 当前时间
    this.aesKey = null // AES密钥
    this.sessionIdAndin = null // 会话编号Andin查询
    this.accountIdAndeq = null // 账号编号Andeq查询
    this.loginKeyAndeq = null // 登录标识Andeq查询
    this.accountIdAndKeyLike = null // 账号编号AndKeyLike查询
    this.loginKeyOrKeyLike = null // 登录标识OrKeyLike查询
    this.aesKeyOrKeyLike = null // AES密钥OrKeyLike查询
  }
}
